services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -2651,6 +2651,7 @@
                 "id": "qualityMeasure_plan_measureResults",
                 "type": "text",
                 "validation": "text",
+                "repeat": "plans",
```

Comment:
Request: Not related to this field but I have nowhere else to put it. At design's request, Scott has changed the validation of the field with id `qualityMeasure_nqfNumber` from `number` to `text`. Might as well change it in this PR too so no one is confused during review. 🤷